guide/locations/provisioned-machine-requirements.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Provisioned Machine Requirements
+layout: website-normal
+---
+
+The requirements for how a provisioned machine should behave will depend on the
+entites subsequently deployed there.
+
+Below are a set of common assumptions, made by many entity implementations, which
+could cause subsequent errors if they do not hold. These relate to the machine's 
+configuration, rather than additional networking or security that a given Cloud 
+might offer.
+
+Also see the [Troubleshooting]({{ site.path.guide }}/ops/troubleshooting/) docs.
+
+
+## Remote Access
+
+### SSH or WinRM Access
+
+Many entities require ssh'ing (or using WinRM for Windows), to install and configure 
+the software.
+
+An example of disabling all ssh'ing is shown below:
+
+    location:
+      aws-ec2:us-east-1:
+        identity: XXXXXXXX
+        credential: XXXXXXXX
+        waitForSshable: false
+        pollForFirstReachableAddress: false
+    services:
+    - type: org.apache.brooklyn.entity.software.base.EmptySoftwareProcess
+      brooklyn.config:
+        onbox.base.dir.skipResolution: true
+        sshMonitoring.enabled: false
+
+
+### Parsing SSH stdout: No Extra Lines
+
+For entities that execute ssh commands, these sometimes parse the resulting stdout.
+
+It is strongly recommended that VMs are configured so that no additional stdout is written when executing 
+remote ssh (or WinRM) commands. Such stdout risks interfering with the response parsing in some blueprints.
+
+For example, if configuring the VM to write out "Last login" information, this should be done for only 
+"interactive" shells (see [Stackoverflow](http://stackoverflow.com/a/415444/1393883) for more details).
+
+
+### Passwordless Sudo
+
+Does passwordless sudo work?
+
+Try executing:
+
+    sudo whoami
+
+See [Passwordless Sudo]({{ site.path.guide }}/locations/index.html#passwordless-sudo).
+
+
+## Advertised Addresses
+
+### Hostname Resolves Locally
+
+Does the hostname known at the box resolve at the box?
+
+Try executing:
+
+    ping $(hostname)
+
+if not, consider setting `generate.hostname: true` in the location config, for jclouds-based locations.
+
+
+### IP Resolves Locally
+
+For the IP address advertised in Brooklyn using the sensor `host.addresses.private` (or `host.subnet.address`), 
+can the machine reach that IP?
+
+Get the sensor value, and then try executing:
+
+    ping ${PRIVATE_IP}
```

Comment:
You should indicate that these commands should be run on the target machine and It wont work locally.